src/condor_starter.V6.1/starter_v61_main.cpp

```diff
@@ -164,6 +166,13 @@ printClassAd( void )
 	if (htcondor::Singularity::enabled()) {
 		printf("%s = True\n", ATTR_HAS_SINGULARITY);
 		printf("%s = \"%s\"\n", ATTR_SINGULARITY_VERSION, htcondor::Singularity::version());
+
+		if (htcondor::Singularity::canRunSandbox())  {
+			printf("%s = True\n", ATTR_HAS_SANDBOX_IMAGE);
+		}
+		if (htcondor::Singularity::canRunSIF())  {
+			printf("%s = True\n", ATTR_HAS_SIF);
+		}
```

Comment:
This all well and good, but don't we need to advertise `ATTR_HAS_CONTAINER` here too?

Reply:
agreed